docs/dc-apps-performance-toolkit-user-guide-bitbucket.md

```diff
@@ -339,6 +339,31 @@ To check status of indexing:
 If case of any difficulties with Index generation, contact us for support in the [community Slack](http://bit.ly/dcapt_slack) **#data-center-app-performance-toolkit** channel.
 {{% /note %}}
 
+## Execution environment
+
+There are two options for running DC App Performance Toolkit:
+ - Run Toolkit on your local machine.  It is recommended for spiking, debugging and custom actions developing. 
+ - Run Toolkit on the performer instance (dedicated AWS EC2 machine).  It is strongly recommended to use performer instance for submission process results generation.
+ In comparison with local execution running Toolkit on the performer instance provides fixed CPU, memory and network performance which guarantees stable and accurate results.
+ ### Local execution environment set up
+ Recommended hardware requirements:
+ - Supported platforms: Linux, MacOS, Windows
+ - CPU: 4 cores
+ - Memory: 16 GB
+
+To set up local execution environment follow [README.md](https://github.com/atlassian/dc-app-performance-toolkit) instructions.
+ 
+ ### Performer instance execution environment set up
+ 1. [Launch AWS EC2 instance](https://docs.aws.amazon.com/quickstarts/latest/vmlaunch/step-1-launch-instance.html). Instance type: [`c5.2xlarge`](https://aws.amazon.com/ec2/instance-types/c5/), OS: select from Quick Start `Ubuntu Server 18.04 LTS`.
+ 2. Connect to the AWS EC2 instance via SSH and [install Docker](https://docs.docker.com/engine/install/ubuntu/#install-using-the-repository).
+ 3. Get your code to the performer instance:
+    3. Create fork of [dc-app-performance-toolkit](https://github.com/atlassian/dc-app-performance-toolkit).
+    3. Clone your fork locally, make changes in the `bitbucket.yml` configuration file and other files if needed.
+    3. Push your changes to the forked repository.
+    3. Clone repository fork.
```

Comment:
Why are you cloning the fork twice?

Reply:
Second `Clone repository fork.` - this command needs to be executed on AWS EC2 instance. In other words: create fork -- clone locally and make changes -- push -- clone on the AWS instance with changes.